pandas/tests/arrays/test_datetimelike.py

```diff
@@ -76,6 +76,14 @@ def timedelta_index():
     return TimedeltaIndex(["1 Day", "3 Hours", "NaT"])
 
 
+@pytest.fixture
+def fixed_now_ts():
```

Comment:
This should just be defined once in `pandas/tests/conftest.py` so it doesn't have to be redefined in multiple places.

Also please add non-zero minute, second and microsecond

Reply:
Yes, of course, thanks for the advice.

Used fixed_now_ts where it does not interfere with reading the code.